spec/dynamoid/persistence_spec.rb

```diff
@@ -2293,6 +2293,29 @@ def around_update_callback
       end.not_to change { obj.class.count }
     end
 
+    context 'disable_create_table_on_save' do
+
+      before do
+        Dynamoid.configure do |config|
+          config.create_table_on_save = false
+        end
+      end
+
+      after do
+        Dynamoid.configure do |config|
+          config.create_table_on_save = true
+        end
+      end
```

Comment:
I wouldn't hardcode option's current value and would just save and restore any value that was set before.

I mean the following:

```ruby
      before do
        Dynamoid.configure do |config|
          @create_table_on_save_old = config.create_table_on_save
          config.create_table_on_save = false
        end
      end

      after do
        Dynamoid.configure do |config|
          config.create_table_on_save = @create_table_on_save_old
        end
      end
```